src/Installers/Windows/WindowsHostingBundle/Product.targets

```diff
@@ -7,22 +7,30 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <Platforms Include="x64;x86;arm64" />
-    <RuntimeInstallers Include="$(DepsPath)dotnet-runtime-$(MicrosoftNETCoreAppRuntimeVersion)-win-x64.exe">
+    <RuntimeInstallers Include="$(DepsPath)dotnet-runtime-$(MicrosoftNETCoreAppRuntimeVersion)-win-x64.msi">
       <TargetPlatform>x64</TargetPlatform>
       <BundleNameProperty>DotNetRedistLtsInstallerx64</BundleNameProperty>
       <Version>$(MicrosoftNETCoreAppRuntimeVersion)</Version>
     </RuntimeInstallers>
-    <RuntimeInstallers Include="$(DepsPath)dotnet-runtime-$(MicrosoftNETCoreAppRuntimeVersion)-win-x86.exe">
+    <RuntimeInstallers Include="$(DepsPath)dotnet-runtime-$(MicrosoftNETCoreAppRuntimeVersion)-win-x86.msi">
```

Comment:
We need two additional MSIs for runtime, similar to what we do in desktop. Both the host and hostfxr MSIs need to be included

Reply:
Added

Reply:
Also realized we don't need the ProductSearch anymore, removed those too